app/blueprints/api.py

```diff
@@ -24,3 +24,20 @@ def sheet():
 
     results = entries.get()
     return jsonify([result.to_dict() for result in results])
+
+
+@api_blueprint.route('/locations', methods=['GET'])
+def locations():
+    db = api_blueprint.config['DB']
+    db_name = os.environ['DB_NAME']
+
+    entries = db.collection(db_name).get()
+
+    locations = []
```

Comment:
@kierana this can probably be a set to begin with 

```suggestion
    locations = set()
```